### ISSUE TITLE
joomla-4 local environment setup 

### ISSUE THREAD

**Issue by Ankur-Khandelwal** [joomla/joomla-cms#32648]
### Problem identified
I was trying to setup the local environment for joomla-4. I followed the instructions mentioned here https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment. I cloned the joomla-cms repo and switched to 4.0-dev branch. Then I copied the entire folder into xampp. And then on running the same on the browser, I am getting the following screen. 
![joomla-install-error](https://user-images.githubusercontent.com/55295915/110841861-6e46b900-82cc-11eb-9dc6-919f0785fb90.png)

### Open questions
Can you tell me if I am missing some steps ?

**Comment by drmenzelit** [comment]
That is not an issue, so I will close this after commenting. 
You haven't followed all steps from the documentation. You need to run composer install and npm ci to get the setup complete. Please re-read the documentation and use the forum or, if you are applying for GSoC, the glip channel for students for further questions. Thanks

**Comment by Ankur-Khandelwal** [comment]
Hi, @drmenzelit. Thanks for replying. 

Actually, I did run composer install as well as npm ci.  I am getting the following error:
`npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! joomla@4.0.0 build:js: `node build/build.js --compile-js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the joomla@4.0.0 build:js script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above. 
`

Yes, I am applying for GSoC and I have dropped a mail at jgsoc@googlegroups.com for the glip channel invite, and I am waiting for the same.

Thank You.
